.github/workflows/main.yml

```diff
@@ -126,8 +126,10 @@ jobs:
             ${{ runner.os }}-sccache-v10-
       - name: Build macOS installable archive
         run: |
-          sudo xcode-select --switch /Applications/Xcode_12.app/Contents/Developer/
+          sudo xcode-select --switch /Applications/Xcode_12.2.0.app/Contents/Developer/
           ./utils/webassembly/ci.sh
+        env:
+          SKIP_XCODE_VERSION_CHECK: 1
```

Comment:
Xcode 12.2 Beta 3 is going to be included in GHA images soon after https://github.com/actions/virtual-environments/pull/1851 is merged, so maybe we should wait until then and turn off the check?

Reply:
I want to merge this as soon as possible to avoid making diff large if CI will pass before actions/virtual-environments#1851 merged. And I'm not sure the virtual-environments PR will be merged quickly.